src/Aws/Lambda/Runtime.hs

```diff
@@ -190,6 +224,10 @@ publishError Context {..} lambdaApiEndpoint (InvocationError err) = do
   let endpoint = "http://"<> lambdaApiEndpoint <> "/2018-06-01/runtime/invocation/"<> awsRequestId <> "/error"
   void (liftIO $ Wreq.post (toString endpoint) (encodeUtf8 @Text @ByteString err))
 
+publishError Context {..} lambdaApiEndpoint (ParseError t t2) = do
+  let endpoint = "http://"<> lambdaApiEndpoint <> "/2018-06-01/runtime/invocation/"<> awsRequestId <> "/error"
```

Comment:
@NickSeagull I'm a bit out of context but what this date `2018-06-01` stands for? It's duplicated in other lines so I'm wondering if that should be a kind of config param

Reply:
It's an API version of AWS, but you are right, I should abstract it somehow. Thanks!